tests/p2p/test_sync_mempool.py

```diff
@@ -80,26 +80,19 @@ def test_mempool_basic(self):
             dot1 = GraphvizVisualizer(self.manager1.tx_storage, include_verifications=True, include_funds=True).dot()
             dot1.render('mempool-test')
 
-        manager2 = self.create_peer(self.network, enable_sync_v1=True)
-        self.assertEqual(manager2.state, manager2.NodeState.READY)
+        self.manager2 = self.create_peer(self.network, enable_sync_v1=True)
+        self.assertEqual(self.manager2.state, self.manager2.NodeState.READY)
 
-        conn = FakeConnection(self.manager1, manager2)
+        conn = FakeConnection(self.manager1, self.manager2)
         for _ in range(1000):
             if conn.is_empty():
                 break
             conn.run_one_step(debug=True)
             self.clock.advance(1)
 
         self.assertConsensusValid(self.manager1)
-        self.assertConsensusValid(manager2)
-        self.assertConsensusEqual(self.manager1, manager2)
-
-        # 3 genesis
-        # 25 blocks
-        # Unlock reward blocks
-        # 8 txs
-        self.assertEqual(len(manager2.tx_storage.indexes.mempool_tips.get()), 1)
```

Comment:
Isn't it better to add a conditional here (e.g., `if manager2.tx_storage.indexes.mempool_tips is not None`) instead of moving it to the sync-v2 test?

Reply:
Actually I'll change this to use the implementation that uses the tx-tips.

Reply:
Thanks!